tests/support/pscide/src/RebuildSpecWithHiddenIdent.purs

```diff
@@ -1,4 +1,6 @@
 module RebuildSpecWithHiddenIdent (exported) where
 
 hidden x _ = x
+
+exported :: forall a. a -> a
```

Comment:
@paf31 @garyb This is the change that fixes the hanging. Without it `psc` gets stuck. I can't reproduce it on the commandline though.... it only happens in `compile` in `tests/Language.PureScript.Ide.RebuildSpec`.